docs/docs/page-build-optimizations-for-incremental-data-changes.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Page Build Optimizations for Incremental Data Changes
+---
+
+Gatsby sources data from multiple sources (CMS, static files - like Markdown, databases, APIs, etc) and creates an aggregated dataset in GraphQL. Currently, each gatsby build uses the GraphQL dataset and queries to do a complete rebuild of the whole app - ready for deployment - including static assets like HTML, JavaScript, JSON, media files, etc.
```

Comment:
```suggestion
Gatsby sources data from multiple sources (CMS, static files - like Markdown, databases, APIs, etc) and creates an aggregated dataset in GraphQL. Currently, each `gatsby build` uses the GraphQL dataset and queries to do a complete rebuild of the whole app - ready for deployment - including static assets like HTML, JavaScript, JSON, media files, etc.
```
If you meant Gatsby build instead of the command that's fine as well.

Reply:
Updated to capital 'G' Gatsby